### PR TITLE
README.md: Add GIF for coala in atom

### LINKED ISSUE #20: Add gif file to show usage
Like https://atom.io/packages/atom-css-comb

### PR DESCRIPTION
**Pull request by yashovardhan**
I have added the desired GIF in the README file. Please have a view.

Closes https://github.com/coala/coala-atom/issues/20

### CODE REVIEW

**Comment by yashovardhanagrawal** [comment]
@AbdealiJK @sils @RaiVaibhav @SanketDG 

you people may have view this PR 😄

**Comment by Makman2** [comment]
Please see our commit guidelines ;)

**Comment by yashovardhan** [comment]
@Makman2  sorry for that

I deleted the comment thinking it was not appropriate to ask for review

**Comment by yashovardhan** [comment]
Why is CircleCI failing here?

I cannot understand

**Review by prnvdixit — CHANGES_REQUESTED** [review]
@yashovardhanagrawal Please adhere to [Commit Guidelines](https://coala.io/commit).
1. Your Shortlog isn't following the guidelines. 😄
2. Please, add a issue reference too. 😃 👍

**Review by soumya-ranjan7 — COMMENTED** [review]
CircleCi failed because of your commit message I guess.

**Review by RJ722 — CHANGES_REQUESTED** [review]
circleci says:
```
Project wide:
**** GitCommitBear [Section: Git | Severity: NORMAL] ****
!    ! Shortlog of HEAD commit isn't in imperative mood! Bad words are 'Added'
```
A better commit message would be `Add GIF in README.md` 
@yashovardhanagrawal use `git commit --amend` to edit the commit message.
Also, you may want to refer to http://api.coala.io/en/latest/Developers/Git_Basics.html

**Comment by RJ722** [comment]
Also, can you add something which gets corrected automatically, like usage of wrong quotation marks "" -> ''?

Makes it look more appealing IMHO 😄

**Comment by yashovardhan** [comment]
@RJ722 I have edited the commit message

can you specify what you want me to add?

**Comment by RJ722** [comment]
I was actually referring to the `gif`. Can you add something which differentiates coala from other linters like analysing two different files - one may be a popular one, like python. The other one may be coffee.

On a related note, you can show the analysis of 'QuotesBear'.
> Also, can you add something which gets corrected automatically, like usage of wrong quotation marks "" -> ''?

**Comment by yashovardhan** [comment]
@RJ722 will do that for sure, (done that partially)

Also, can you tell me how you view the error in CircleCI, is there any tutorial that I can find about it specific to coala?

I wanted to be as good as possible before GCI happens 😛

**Comment by Mixih** [comment]
commit body is still too long

**Review by prnvdixit — CHANGES_REQUESTED** [review]
@yashovardhanagrawal 
1. Your Shortlog should ideally be ```README.md: Add GIF for coala in atom``` 😄 👍
2. Also, CI is failing due to inconsistencies in Markdown Bear - Please run ```coala``` on changed file before committing 😄 
3. Opening the GIF gives this [error](https://ibb.co/j2LSfm) 😃

**Comment by yashovardhan** [comment]
@prnvdixit 
1. I edited my Shortlog
2. I ran the test for Markdownbear, its working fine there, but failing here I don't understand why.
3. You may check the GIF now

**Comment by yashovardhan** [comment]
```
Executing section CoffeeScript...
Executing section PYTHON...
Executing section JSON...
Executing section Markdown...
Executing section Git...
[nltk_data] <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED]
[nltk_data]     certificate verify failed (_ssl.c:777)>
[WARNING][06:13:05] Bear GitCommitBear failed to run. Take a look at debug messages (`-V`) for further information.
Executing section cli...
```

**Comment by yashovardhan** [comment]
@prnvdixit also that error in GIF you are pointing about will be solved after this PR is merged

it is happening because the GIF is still present in my fork and not it the master branch of coala-atom, hence 404 error is coming while trying to open that gif from any external link

**Comment by jayvdb** [comment]
```
[nltk_data] <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED]
[nltk_data]     certificate verify failed (_ssl.c:777)>
```

That is `GitCommitBear` being strange.
Probably a different bug, and it will probably magically fix itself after a while.
But if you want to investigate it, the nltk data can be fetched manually using `python -m nltk.downloader`

**Comment by prnvdixit** [comment]
@yashovardhanagrawal IMO GIF error shouldn't had been there due to relative location addressing..... It's fine now 👍
Also, do include changes suggested by Rahul 😄

**Comment by yashovardhan** [comment]
@jayvdb so what should I do? the error is still there.
I tried nltk_data same SSL error was there...

**Comment by yashovardhan** [comment]
@prnvdixit I incorporated changes suggested by @RJ722 by implementing the GIF across python and CoffeeScript both. The QuotesBear thing cannot be implemented in atom I guess, because I tried it with terminal it was working fine but not showing the error in atom.

**Comment by jayvdb** [comment]
@yashovardhanagrawal , please squash your commits

**Comment by yashovardhan** [comment]
@jayvdb earlier this was done in 1 commit, I tried to check whether the CI fails in a specific commit that's why I separated it.

squashed it again anyway

**Comment by jayvdb** [comment]
Thanks.  We've also got a Circle breakage, which I am working on https://github.com/coala/coala-atom/issues/81

**Comment by yashovardhan** [comment]
@jayvdb thanks a lot 😄

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-atom/pull/75/commits/e5e89cd6208e11973e4dc8e6f9d7083fc969a5da

**Comment by jayvdb** [comment]
Unfortunately I cant finish #81 now.
Go on with a `low` issue, but in a different repository ;-)

**Comment by blazeu** [comment]
Please do a rebase.
https://github.com/coala/coala-atom/issues/81 is fixed.

**Comment by yashovardhan** [comment]
@blazeu its still failing 😕

**Comment by blazeu** [comment]
Your [circle.yml](https://github.com/yashovardhanagrawal/coala-atom/blob/cf1b58c6ee2be37f3f4a55a90c124ed8c8fff875/circle.yml) is still old.

**Comment by yashovardhan** [comment]
@blazeu I updated it, its still failing

**Comment by yashovardhan** [comment]
@blazeu @jayvdb 

just for sake of experimenting to find out what the problem is with CI, I removed my commit from README.md.
Even then, it's failing with the error 
`**** MarkdownBear [Section <empty> | Severity NORMAL] ****
!    ! The text does not comply to the set style.
[----] /home/ubuntu/coala-atom/README.md
[++++] /home/ubuntu/coala-atom/README.md`

that too is the file which is not edited. (README.md)

Locally no such type of problem is existing. 

Anyhow, I am restoring my commit in README.md 😛

**Comment by blazeu** [comment]
Unfortunately https://github.com/coala/coala-atom/issues/81 is reopened.

File: circle.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+machine:
+  node:
+    version: 6.11.5
```
Comment (Makman2) [inline circle.yml]: shouldn't this go into a separate commit? ;)

Comment (blazeu) [reply circle.yml]: And that is my PR https://github.com/coala/coala-atom/pull/83
He didn't rebase properly

**Comment by jayvdb** [comment]
@yashovardhanagrawal , please do not close valid pull requests.

**Review by jayvdb — APPROVED** [review]

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by sils** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by sils** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:
